session-1-Local-Variable-Type-Inference/02_Object_Data_Types/README.md

```diff
@@ -0,0 +1,28 @@
+**Local Variable Type Inference for Object Data Types example**
```

Comment:
Apply same corrections from the above to this section as well

Reply:
Done